docs/agents/holo_synth.rst

```diff
@@ -0,0 +1,65 @@
+.. highlight:: rst
+
+.. _holo_synth_agent:
+
+==============
+Holography Synthesizer Agent
+==============
+
+The Holography Synthesizer Agent is provided with OCS to help demonstrate and debug issues with the holography synthesizers, which provide a signal at a desired frequency for holography measurements.
+It will connect the computer to the two Local Oscillators (LO's) via USB port, initialize the LO's, set the frequency of each and pass the frequency to the OCS feed.
+
+.. argparse::
+   :module: agents.holo_synth.holo_synth_agent
+   :func: make_parser
+   :prog: synth_agent.py
+
+Dependencies
+------------
+
+Python Packages: 
+- holo_daq
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+````````````````
+
+To configure the Holography FPGA Agent we need to add a FPGAAgent block to our
+ocs configuration file. Here is an example configuration block using all of the
+available arguments::
+
+      {'agent-class': 'SynthAgent',
+       'instance-id': 'synth_LO',
+       'arguments': [['--config_file','holog_config.yaml']]}
+
+Description
+-----------
+
+The SynthAgent contains functions which control the two synthesizers for holography measurements.  Before the synthesizers can output a frequency, the user needs to initialize both using the init_synth() function.  This will connect to the 2 synthesizers via 2 USB ports and prepares them to read in the user-desired frequency as the signal output.
+
+Once the synthesizers are initialized, the user can take set the frequency output using the set_frequencies() function.  This will set the frequency output of BOTH synthesizers.  The user-specified frequency should be in GHz
+
+Subsection
+``````````
+
+# Use subsections where appropriate.
+
+Agent API
+---------
+
+.. autoclass:: agents.holo_fpga.holo_fpga_agent.FPGAAgent
+    :members:
+
+Example Clients
+---------------
+
+To initialize the ROACH2 FPGA, use the "fpga" agent::
+
+    from ocs.ocs_client import OCSClient
+    agent_fpga = OCSClient("fpga")
+    agent_fpga.init_FPGA()
```

Comment:
This example looks identical to the one in `holo_fpga.rst`. Either update with example for `holo_synth` agent or remove this section.

Reply:
Still identical to the holo fpga docs page. I'd change this to:

```
from ocs.ocs_client import OCSClient
client = OCSClient("synth_LO")
client.init_synth()
```

And modify the leading text to reference the synth, rather than the roach2.

Reply:
Oops, fixed.